website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -398,6 +398,61 @@ function MyPlayground(props) {
 
 </BrowserWindow>
 
+Or with [`import` statement syntax](https://github.com/nihgwu/react-runner#import-statement-and-multi-files)
+
+```jsx title="src/theme/ReactLiveScope/index.js"
+import React from 'react';
+import {createRequire} from 'react-runner';
+
+// highlight-start
+const ButtonExample = (props) => (
+  <button
+    {...props}
+    style={{
+      backgroundColor: 'white',
+      color: 'black',
+      border: 'solid red',
+      borderRadius: 20,
+      padding: 10,
+      cursor: 'pointer',
+      ...props.style,
+    }}
+  />
+);
+// highlight-end
+
+// Add react-live imports you need here
+const ReactLiveScope = {
+  React,
+  ...React,
+  // highlight-start
+  require: createRequire({
+    './ButtonExample': ButtonExample,
+  }),
+  // highlight-end
+};
+
+export default ReactLiveScope;
```

Comment:
shouldn't we change the name of this component to something like `LiveCodeBlockComponents` ? 🤷‍♂️ 
maybe later